src/LoginCommand.java

```diff
@@ -1,7 +1,8 @@
 
 public class LoginCommand {
-    public void execute(String userName, String userPassword) {
-        if (UserContainer.isUserNameAndPasswordExists(userName, userPassword)) {
+    public void execute(String userName, String Password) throws UserNotFoundException {
```

Comment:
Why did you change `userPassword` param name to `Password`
it is ok to name it `password`, but do not use capital letters for variables

Reply:
Do you need throws here?

Reply:
it is an accident